src/layout/Cell.js

```diff
@@ -19,16 +19,29 @@
     Cell.prototype.publish("gridRowSpan", 1, "number", "Grid Row Span",null,{tags:["Private"]});
     Cell.prototype.publish("gridColSpan", 1, "number", "Grid Column Span",null,{tags:["Private"]});
     Cell.prototype.publish("handleSize", 6, "number", "Grid Row Position",null,{tags:["Private"]});
+    
+    Cell.prototype.publish("indicateTheseIds", [], "array", "Array of DOM Ids to display update-indicators over.",null,{tags:["Private"]});
+    
+    Cell.prototype.publish("indicatorColor", "#FF00FF", "html-color", "Background color of update-indicator",null,{tags:["Basic"]});
+    Cell.prototype.publish("indicatorOpacity", 0.2, "number", "Opacity of update-indicator",null,{tags:["Basic"]});
 
     Cell.prototype.enter = function (domNode, element) {
         Surface.prototype.enter.apply(this, arguments);
         element.classed("layout_Surface", true);
+        
+        //Example values...
+        this.indicateTheseIds(["_w43","_w59"]);
+        
```

Comment:
Can you move this into its own Test in test/layoutFactory - I suspect the issue I think I am seeing is becuase this is being set for every cell...